printer_mods/roboticator24/side-mount_spoolholder/README.md

```diff
@@ -0,0 +1,30 @@
+![in position](./img/inposition.jpg)
+
+# Side-Mount Spoolholder
```

Comment:
This mod is a duplicate or pull request 220 https://github.com/VoronDesign/VoronUsers/pull/220 .  We try to avoid duplicates in the VoronUsers repo - is there any advantage to your mod over the one in PR 220?  If not, and you still want to merge the rest of your PR, you can remove this horizontal spool holder mod from the PR.